README.md

```diff
@@ -1459,18 +1432,117 @@ this in mind, an implementation in IPLD/IPFS is favored:
     "share": "1, 2, 3, ..., 100",
     "validFrom": { "/": "<hash pointing to RDF-Schema of Date" },
     "validTo": { "/": "<hash pointing to RDF-Schema of Date" },
-    "manifestation": { "/": "<hash pointing to the a Manifestation>" },
-    "license": { "/": "<hash pointing to the license>" }
+    "source": { "/": "<hash pointing to a Copyright>" },
+    "license": { "/": "<hash pointing to a license>" }
 }
 ```
 
+In our transformation, it is important to highlight that every `Right` must include a `source` (or
+equivalent) property that links it to an enabling `Copyright` or parent `Right`. With use on an
+immutable ledger in mind, the `source` property implements support for `lcc:SourceRight`s–albeit in
+a "backwards" relation in comparison to the RRM's definition–a `Right` containing a `source`
+property is the derivation while the pointed-to entity is the `lcc:SourceRight`.
+
+#### Copyright Semantics
+
+Although RRM `Right`s are capable of representing both full copyrights as well as derived licenses
+to `Creation`s, we split these two concepts into different entities to better represent them within
+distributed ledgers. We base the structure of the `Copyright` entity on the `Right` entity's, but as
+only a subset of the `Right`'s properties pertain to `Copyright`s (e.g. "territory", "validFrom",
+etc.), we do not require implementations to subtype `Copyright`s from `Right`s. However,
```

Comment:
The highlighting is funky here, but:

> as only a subset of the `Right`'s properties pertain to `Copyright`s

I think, logically, it would make sense for `Copyright`s to be a subtype of `Right`, but inheriting all the properties is also a pain. The current data models have them as sibling subtypes of `schema:Intangible`.